paddle/fluid/pybind/eager_method.cc

```diff
@@ -1863,6 +1917,33 @@ static PyObject* tensor_method_get_non_zero_indices(TensorObject* self,
   EAGER_CATCH_AND_THROW_RETURN_NULL
 }
 
+PyDoc_STRVAR(tensor_method_values__doc__,
+             R"DOC(values($self, /)
+--
+
+**Notes**:
```

Comment:
```suggestion
Note:
```
另外，这个api方法的文档好像在python端已经存在..，不过c++端添加了好像也没什么问题